src/LibraryManager.Vsix/UI/Controls/TargetLocation.xaml.cs

```diff
@@ -279,5 +290,26 @@ private void TargetLocation_LostFocus(object sender, RoutedEventArgs e)
                 Flyout.IsOpen = false;
             }
         }
+
+        private void LibraryNameChanged(object sender, PropertyChangedEventArgs e)
+        {
+            string targetLibrary = _libraryNameChange.LibraryName;
```

Comment:
>targetLibrary [](start = 19, length = 13)

Check this string to be null or empty.

Reply:
Will do

---
In reply to: [201086028](https://github.com/aspnet/LibraryManager/pull/285#discussion_r201086028) [](ancestors = 201086028)